Telegram Stickers/Readme

```diff
@@ -0,0 +1,8 @@
+Stickers uploaded will be reviewed before they are uploaded and made visible in the THORChain Telegram Stickers pack.
```

Comment:
@THORChainMike can you rename the file to README.md and then add an h1 title to the the top of the document?

Reply:
Should be okay now.